dev/tools/shared/images.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "service_name": "",
+    "context_dir": ""
```

Comment:
I'd recommend being explicit here, as it's not clear what this refers to. Something like:

```
"service_name": "agent-sandbox-controller",
"context_dir": ".",
"dockerfile": "Dockerfile"
```